### PR TITLE
Issue/2536 device list script (Connect #2536)

### PR DESCRIPTION
**Pull request by stellanl**
## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: scripts/data/src/org/akvo/gae/remoteapi/FixDeviceFileJobQueue.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     }
+
+        
+    private void processDeviceFileJobQueue(DatastoreService ds) throws InterruptedException {
+
+        System.out.println("#Processing DeviceFileJobQueue");
+        
+        //find all jobs made since we changed from wfpPhotonnnn format
+        final Filter f = new FilterPredicate("fileName", FilterOperator.LESS_THAN, "wfp"); 
+        final Filter f2 = new FilterPredicate("fileName", FilterOperator.GREATER_THAN, "ff");
```
Comment (muloem) [inline scripts/data/src/org/akvo/gae/remoteapi/FixDeviceFileJobQueue.java]: The two filters are not used so maybe just remove them?

File: scripts/data/src/org/akvo/gae/remoteapi/FixDeviceFileJobQueue.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        final Query group_q = new Query("DeviceFileJobQueue");//.setFilter(f).setFilter(f2);
```
Comment (muloem) [inline scripts/data/src/org/akvo/gae/remoteapi/FixDeviceFileJobQueue.java]: Filters here are commented out.

File: scripts/data/src/org/akvo/gae/remoteapi/CheckTrimDevicesAndDeviceFiles.java [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/*
+ * - Checks that all surveys, groups, questions and options are consistent
+ */
+public class CheckTrimDevicesAndDeviceFiles implements Process {
```
Comment (muloem) [inline scripts/data/src/org/akvo/gae/remoteapi/CheckTrimDevicesAndDeviceFiles.java]: This looks like a different datascript altogether..?

Comment (stellanl) [reply scripts/data/src/org/akvo/gae/remoteapi/CheckTrimDevicesAndDeviceFiles.java]: It must have slipped into the commit...

**Pull request merged**